src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const changedRobot = {};
+
+  for (const key in robot) {
+    const newValue = robot[key];
+
+    if (!changedRobot.hasOwnProperty(newValue)) {
+      changedRobot[robot[key]] = key;
+    } else {
+      return null;
+    }
```

Comment:
[checklist](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md) 4th item

Reply:
this else stay in for so if I deleate else null will return every time